adapters/src/chainlink.ts

```diff
@@ -52,9 +52,9 @@ async function fetchAsset (symbol: string, provider?: any): Promise<AssetPrice>
   return newAssetPrice(symbol, price, 'USD', (new BigNumber(timestamp.toString())).multipliedBy(1000))
 }
 
-export default async function (symbols: string[], _apiToken?: string): Promise<AssetPrice[]> {
-  const apiToken = process.env.API_TOKEN ?? _apiToken
+export default async function (symbols: string[], apiToken: string): Promise<AssetPrice[]> {
```

Comment:
```suggestion
export default async function (symbols: string[], apiToken?: string): Promise<AssetPrice[]> {
```
perhaps change back to optional?? as the apiToken is not mandatory on InfuraProvider

Reply:
Oh, Joel can you try to make the test run on the actual endpoint if that's the case.